src/Components/CreateImageWizardV2/steps/Snapshot/Snapshot.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+import {
+  Alert,
+  Button,
+  DatePicker,
+  Flex,
+  FormGroup,
+  Grid,
+  Radio,
+  Text,
+  Title,
+} from '@patternfly/react-core';
+
+import ConditionalTooltip from './components/ConditionalTooltip';
+
+import { useListFeaturesQuery } from '../../../../store/contentSourcesApi';
+import { useAppDispatch, useAppSelector } from '../../../../store/hooks';
+import {
+  selectSnapshotDate,
+  selectUseLatest,
+  changeUseLatest,
+  changeSnapshotDate,
+} from '../../../../store/wizardSlice';
+import {
+  dateToMMDDYYYY,
+  parseMMDDYYYYtoDate,
+} from '../../../../Utilities/time';
+
+const dateValidators = [
+  (date: Date) => {
+    if (date.getTime() > Date.now()) {
+      return 'Cannot set a date in the future';
+    }
+    return '';
+  },
+];
+
+export default function Snapshot() {
+  const dispatch = useAppDispatch();
+  const snapshotDate = useAppSelector(selectSnapshotDate);
+  const useLatest = useAppSelector(selectUseLatest);
+  const setUseLatest = (val: boolean) => dispatch(changeUseLatest(val));
+  const setSnapshotDate = (val: string) => dispatch(changeSnapshotDate(val));
+
+  // TODO: Remove query once snapshotting is no longer a "feature flag" i.e. when in Prod-stable
+  // Ensure to remove the Alert and ConditionalTooltip below.
+  const { data, isSuccess, isFetching, isError } = useListFeaturesQuery();
+  const isSnapshotDisabled =
+    isError ||
+    isFetching ||
```

Comment:
When do we expect this to hit prod-stable? Should we consider putting this step behind an Unleash feature flag? We would like to get the V2 wizard into prod-preview next week with prod-stable to follow shortly thereafter.

Reply:
In prod-stable the other toggle will simply be disabled for now, on hover of the disabled "Use a snapshot" Radio you will see "Snapshotting is not currently available". 

If this is acceptable until snapshotting is live in Prod-stable, then I'd say that you already have your feature flag handled.

Reply:
Would it be available in prod-preview? UX has developed some patterns for us where features are disabled in prod-stable but users are encouraged to check them out in prod-preview. We could try that if this is available in prod-preview (in a follow-up PR).

Reply:
Okay I added some better logic to handle this, it is ugly, but temporary. 
Attempting to incorporate redux-like storage would make pinging the features api much more complex, I would like to avoid that for what should be a very simple change.

Reply:
<img width="695" alt="Screenshot 2024-04-22 at 10 34 22 PM" src="https://github.com/osbuild/image-builder-frontend/assets/38083295/7fb75606-b117-4761-9da5-967326b2b973">